infra/k8s/README.md

```diff
@@ -0,0 +1,68 @@
+# Running Testground with Kubernetes on Digitial Ocean backend
+
+## Requirements
+
+1. Digital Ocean account
+2. [Docker Hub](https://hub.docker.com/) account
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+4. Docker
+5. [helm](https://github.com/helm/helm)
+
+## Background
+
+This document is explaining how to execute a Testground test plan locally, while Testground targets a Kubernetes cluster on Digital Ocean for execution of all tasks. Images are built locally by Testground via Docker, and pushed to Docker Hub. Redis is to be installed manually on Kubernetes, prior to running any test plans.
+
+## Setup Kubernetes Cluster on Digital Ocean
+
+1. Create a Kubernetes cluster on Digital Ocean, with the necessary number of workers.
+
+Note: There is a limit of 100 pods per worker, so calculate the number of workers you need accordingly.
+
+2. Configure your `kubectl` context and authentication with Digital Ocean. Confirm that `kubectl get nodes` shows the number of nodes in your cluster.
```

Comment:
I'm reviewing this from the POV of a Kubernetes noob (me!): am I supposed to run this in DO or locally?

Reply:
`kubectl` is a control tool for Kubernetes, that you run locally. It has to be configured with any given Kubernetes cluster.

This section is sparse on purpose because configuring `kubectl` with a cluster is mostly different for every single provider - AWS, DO, GCP, etc.

Reply:
@hacdias I added an `Introduction` section with some basic intro to the requirements/tools, let me know if you have any questions, and it you think the descriptions now are enough for people with very little Kubernetes experience :)